chat-widget/src/components/headerstateful/HeaderStateful.tsx

```diff
@@ -34,7 +34,7 @@ export const HeaderStateful = (props: IHeaderStatefulParams) => {
         },
         onCloseClick: async () => {
             TelemetryHelper.logActionEvent(LogLevel.INFO, { Event: TelemetryEvent.HeaderCloseButtonClicked, Description: "Header Close button clicked." });
-            if (conversationState.current === ConversationState.Active) {
+            if (conversationState.current === ConversationState.Active || (conversationState.current === ConversationState.Postchat) && !isDownloadTranscriptDisabled && !isEmailTranscriptDisabled) {
```

Comment:
`!isDownloadTranscriptDisabled && !isEmailTranscriptDisabled`  maynot be combined as both of them are configurable individually.

Reply:
Correct Thanks for the catch we would need them in either case
`!emailTranscriptDisabled || !downloadTranscriptDisabled;`

Reply:
We have to check for both because in either case we have "Get copy of chat" confirmation pane in V1

Reply:
lets keep the same message as it appears before post chat screen.